rustfmt.toml

```diff
@@ -1,2 +1,3 @@
 imports_granularity="Crate"
 format_code_in_doc_comments = true
+edition = "2018"
```

Comment:
Why do we need it?

Reply:
This is for running `rustfmt` without `cargo`

Reply:
Without this, we can use `rustfmt`.